### ISSUE TITLE
Localization of Marketing Data and Categories

### ISSUE THREAD

**Issue by thomasdbruton** [namm-standards/standards#9]
We should add multi-language support for Product names descriptions and NAMM Categories.

**Comment by dan-kessler** [comment]
I assume that you meant to have a comma between names and descriptions?

**Comment by jbkoehler** [comment]
Are we talking about the data or the XML itself?  If just the data, new nodes, or a declarative at the start?

**Comment by thomasdbruton** [comment]
Yes.  Multi-language support for product names, descriptions and NAMM categories.

**Comment by jbkoehler** [comment]
See here: https://www.w3.org/International/questions/qa-when-xmllang.en
